plugin/src/main/java/org/opensearch/ml/utils/TenantAware.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.utils;
+
+import org.opensearch.core.action.ActionListener;
+
+public interface TenantAware {
```

Comment:
Why build this new interface ? I see it's only used by `TenantAwareHelper`, do we really need a new interface ?

Reply:
Agree, we don't need this now. Deleting.